src/diffusers/loaders/lora.py

```diff
@@ -467,25 +492,25 @@ def load_lora_into_unet(
         cls, state_dict, network_alphas, unet, low_cpu_mem_usage=None, adapter_name=None, _pipeline=None
     ):
         """
-        This will load the LoRA layers specified in `state_dict` into `unet`.
+        Load LoRA layers specified in `state_dict` into `unet`.
```

Comment:
Can you provide a code snippet for this @sayakpaul? I couldn't get this to work 😅

Reply:
I think it's okay to skip the code snippet for this given https://github.com/huggingface/diffusers/pull/5813/files#r1396624840. 

We come to code path via `load_lora_weights()` but not the other way around.